### PR TITLE
[doc] naming, Virtual-IP

### PR DESCRIPTION
**Pull request by rfree-d**

### CODE REVIEW

File: doc/hacking.md [hunk 1]
```diff
@@ -24,6 +24,23 @@ To develop the program, we recommend mainly Debian Stable as the main environmen
...
+
+* Virtual-IP - the IP that someone has assigned as result of using Galaxy42 network.
+It often will be in form of IPv6, from some "reserved" block (that is at least unlikelly to be assigned by IANA as part of
+the "regular" Internet).
+
+* Virtual-IP Global - this is an Virtual-IP, where you are the only owner of it, and no one else have such IP even globally,
+usually because it is a hash of public key.
+
+**Open question** - should we use: Virtual-IP, or Galaxy-IP, or Yedino-IP, or Hash-IP here.
+
```
Comment (rfree-d) [inline doc/hacking.md]: It's open question, @da2ce7 how to name "our IP that you get in Galaxy42".

We can start already standardizing naming across our code (while doing error reporting, and while preparing for translations) so have to decide on few such things :)

**Comment by rfree-d** [comment]
Lol -0.008% coverage drop from doc change

File: doc/hacking.md [hunk 2]
```diff
@@ -27,19 +27,16 @@ we're bussy), see contact information in the main README.md of this project.
 # Naming and Glossary
 
 
-* Virtual-IP - the IP that someone has assigned as result of using Galaxy42 network.
-It often will be in form of IPv6, from some "reserved" block (that is at least unlikelly to be assigned by IANA as part of
-the "regular" Internet).
+* Hash-IP - the IP that someone can seldomly own without requiring permission from any authority, because it is derived from public key with some form of Hash.
```
Comment (da2ce7) [inline doc/hacking.md]: Maybe:?

Hash-IP Address: A Virtual IP Address based upon a cryptographic hash. Every Hash-IP is derived from a Public Key. Use of any Hash-IP demands access to the associated Private Key (for that particular Hash-IP Address).

**Comment by rfree-d** [comment]
Updated as https://github.com/yedino/galaxy42/pull/46/files

**Pull request merged**